swagger_spec_validator/common.py

```diff
@@ -11,13 +11,19 @@
 
 import six
 import yaml
+from pkg_resources import resource_filename
 from six.moves.urllib.parse import urljoin
 from six.moves.urllib.request import pathname2url
 from six.moves.urllib.request import urlopen
 try:
     from yaml import CSafeLoader as SafeLoader
-except ImportError:
-    from yaml import SafeLoader
+except ImportError:  # pragma: no cover
+    from yaml import SafeLoader  # pragma: no cover
```

Comment:
nit. No need to have `# pragma: no cover` here (ignoring the `except` line ignores all the underlying block as well
```suggestion
    from yaml import SafeLoader
```